modules/virt-quick-creating-vm.adoc

```diff
@@ -0,0 +1,31 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-create-vms.adoc
+
+:_content-type: PROCEDURE
+[id="virt-quick-creating-vm_{context}"]
+= Quick creating a virtual machine
+
+You can quick create a virtual machine (VM) by using a template with an available boot source.
```

Comment:
I realize that this is in the UI, but this "quick create" phrasing goes against the ISG:

> Do not use a word as a part of speech that is different from its standard dictionary definition. 

Maybe use "quickly" when not referring directly to the UI?